templates/resources.hbs

```diff
@@ -2,46 +2,46 @@
 {{import_globals}}
 <div class="resources-inner">
 	{{{a11y_aria_label _globals._extensions._resources.resources}}}
-	<div class="resources-filter clearfix resources-col-{{return_column_layout_from_collection_length resources '_type'}}">
-		{{! If the resources collection contains only one type of resource (pdf, media or link) don't show anything}}
-		{{#if_collection_contains_only_one_item resources '_type'}}
-		{{else}}
-			<button class="base resources-show-all selected" data-filter="all" aria-label="{{model._filterAria.allAria}}">
-				<span>{{model._filterButtons.all}}</span>
-			</button>
-			{{#if_collection_contains resources '_type' 'document'}}
-			<button class="base resources-show-document" data-filter="document" aria-label="{{model._filterAria.documentAria}}">
-				<span>{{model._filterButtons.document}}</span>
-			</button>
-			{{/if_collection_contains}}
-			{{#if_collection_contains resources '_type' 'media'}}
-			<button class="base resources-show-media" data-filter="media" aria-label="{{model._filterAria.mediaAria}}">
-				<span>{{model._filterButtons.media}}</span>
-			</button>
-			{{/if_collection_contains}}
-			{{#if_collection_contains resources '_type' 'link'}}
-			<button class="base resources-show-link" data-filter="link" aria-label="{{model._filterAria.linkAria}}">
-				<span>{{model._filterButtons.link}}</span>
-			</button>
-			{{/if_collection_contains}}
-		{{/if_collection_contains_only_one_item}}
+	{{#if_has_multiple_types resources}}
+	<div class="resources-filter clearfix resources-col-{{get_column_count resources}}">
+		<button class="base resources-show-all selected" data-filter="all" aria-label="{{model._filterAria.allAria}}">
+			<span>{{model._filterButtons.all}}</span>
+		</button>
+		{{#if_has_type resources 'document'}}
+		<button class="base resources-show-document" data-filter="document" aria-label="{{model._filterAria.documentAria}}">
+			<span>{{model._filterButtons.document}}</span>
+		</button>
+		{{/if_has_type}}
+		{{#if_has_type resources 'media'}}
+		<button class="base resources-show-media" data-filter="media" aria-label="{{model._filterAria.mediaAria}}">
+			<span>{{model._filterButtons.media}}</span>
+		</button>
+		{{/if_has_type}}
+		{{#if_has_type resources 'link'}}
+		<button class="base resources-show-link" data-filter="link" aria-label="{{model._filterAria.linkAria}}">
+			<span>{{model._filterButtons.link}}</span>
+		</button>
+		{{/if_has_type}}
 	</div>
+	{{/if_has_multiple_types}}
 	<div class="resources-item-container" role="list">
 	{{#each resources}}
 	<div class="resources-item drawer-item {{_type}}" role="listitem">
-		<button role="link" class="base resources-item-open drawer-item-open"
+		<a href="{{_link}}" class="base resources-item-open drawer-item-open"
 			data-type="{{_type}}"
-			data-filename="{{filename}}"
-			data-href="{{_link}}"
-			data-force-download="{{_forceDownload}}"
+			{{#if_force_download this}}
+			download="{{filename}}"
```

Comment:
Is it ok that this works for [same-origin links only](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Attributes)? 

Should we perhaps test for cross origin?

Reply:
Good point - but I think since it is just ignored if you try and apply it to different-origin URL it's best dealt with [via documentation](https://github.com/adaptlearning/adapt-contrib-resources/pull/60/commits/72f783e85392dd4b54a530cea369925303923ca3) rather than by technical means

Reply:
Will it not open in the same window if `[download]` is ignored as `[target]` is not defined?

Reply:
that's easily resolved